awx/ui_next/src/components/PaginatedDataList/ToolbarAddButton.jsx

```diff
@@ -28,7 +28,7 @@ function ToolbarAddButton({
       <DropdownItem
         key="add"
         isDisabled={isDisabled}
-        component={linkTo ? Link : Button}
+        component={linkTo ? Link : 'Button'}
```

Comment:
Not sure why this was changed.  It takes a react node, and we want to pass it the PF button.  From their docs.

```
component | React.ReactNode | No | 'a' | A ReactElement to render, or a string to use as the component tag. Example: component={<Link to="/components/alert/">Alert</Link>} Example: component="button" If React.isValidElement(component) the className prop will be injected unless styleChildren="false"
-- | -- | -- | -- | --
```

Reply:
I think you are right